CHANGELOG.rst

```diff
@@ -17,6 +17,9 @@ Summary
 
 Fixed
 ^^^^^
+* Removed the ``ValueError`` raised by :attr:`~.HotRegion.__init__` ``if not phase_bounds or None in phase_bounds`` so that user can set a derived ``phase_shift`` value.
```

Comment:
if you view this file formatted, there will be some Sphinx issues with the cross-referencing I believe.

also general tidying like adding punctuation.